content/markdown/04-intro-to-stats/04-08-linear-regression.Rmd

```diff
@@ -0,0 +1,468 @@
+
+### Interpreation 
+
+Ok, so the important part. Interpretation. Before we even <em>look</em> at the output let's think about this mathematically. Our regression equation here we can think of as this: $$\widehat{weight} = \beta_0 + \beta_1 Sex$$. (Note here that we're using the hat on top of "weight" to indicate it's a predicted not measured value.) So recall that $\beta_0$ is the intercept, so if we find that, and we estimate our $\beta_1$ through regression, we should be able to predict the weight for any new bison that joins the herd, by taking $\beta_0$ and adding $\beta_1$ multiplied by the sex of the new bison. 
+
+But wait. How would we multiply a value "male" or "female". Well, the thing is we can't, and we don't have to! Instead, we use the very clever trick of <strong>indicator variables</strong>, where we assign a unique number to each of the possible categories in our categorical variable. Since we only have two levels in this example, we will use a 0 and 1 scheme. So what we will do, or rather what R will do under the hood, is use a function to indicate whether or not a given animal is a female or not, like this: 
+
+$$I_F = 
+\begin{cases}
+1 & \text{if animal is female}\\
+0 & \text{otherwise}\\
+\end{cases}
+$$ 
+
+Where $I_F$ just stands for Indicator, with the subscript F denoting that we're talking about females. 
+
+So we actually just end up with our regression equation looking like this: $$\widehat{weight} = \beta_0 + \beta_1 I_F$$
+
+where to predict an animal's weight, we'll multiply $\beta_1$ by the appropriate value given by the indicator function above - 0 or 1. 
+
+Now that we've walked through this, let's look at the summary and discuss: 
+
+```{r}
+summary(mod)
+```
+
+So knowing that the indicator function is using females as 0 and males as 1 (we can see this because the coefficient after the intercept is <code>animal_sexM</code>), what would the weight of a female bison be, given no other information?  Well, the formula would be: $$\widehat{weight}_F = \beta_0 + \beta_1 I_F$$ And the indicator function if we look above will be $$I_F = 0$$ so the formula simplifies to: $$\widehat{weight}_F = \beta_0 + \beta_1 * 0$$ and therefore $$\widehat{weight}_F = \beta_0 $$, which is  1001.253.
+
+What about for males? Well, the formula would be: $$\widehat{weight}_M = \beta_0 + \beta_1 I_F$$ And in this case $I_F = 1$ so, that would be after plugging in the values $$\widehat{weight}_M  = 1001.253 + 567.046$$ or in R: 
+
+```{r}
+1001.25 + 567.046
+```
+
+## Multiple Predictor Variables
```

Comment:
Should add a section talking about multicollinearity and producing biplots of all variables that are considered for a model- if any are highly correlated it is possible for lm not to fit